src/main/java/com/feedzai/commons/sql/abstraction/engine/AbstractDatabaseEngine.java

```diff
@@ -57,14 +58,7 @@
 import java.sql.SQLException;
 import java.sql.Statement;
 import java.sql.Types;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Properties;
-import java.util.Set;
+import java.util.*;
```

Comment:
Could you please undo this import, meaning: import only the required classes? Also, do that on other classes where this also happened.
I think you have your Intellij IDEA configured to automatically import using "*" whenever you have "n" number of imports from the same package. "n" seems to be 8 on your case: I think you should increase this value a lot so this doesn't happen.